onnxruntime/server/grpc/prediction_service_impl.cc

```diff
@@ -9,12 +12,29 @@ PredictionServiceImpl::PredictionServiceImpl(const std::shared_ptr<onnxruntime::
 
 ::grpc::Status PredictionServiceImpl::Predict(::grpc::ServerContext* context, const ::onnxruntime::server::PredictRequest* request, ::onnxruntime::server::PredictResponse* response) {
   auto request_id = SetRequestContext(context);
+  // No labels for gRPC counter as we don't have model arguments
+  (*MetricRegistry::Get().totalGRPCRequests)->Add({}).Increment();
+  // Log Request size
+  (*MetricRegistry::Get().grpcRequestSize)->
+    Add({}, MetricRegistry::ByteBuckets()).
+    Observe(request->ByteSize());
   onnxruntime::server::Executor executor(environment_.get(), request_id);
+  auto begin = std::chrono::high_resolution_clock::now();
   //TODO: (csteegz) Add modelspec for both paths.
   auto status = executor.Predict("default", "1", *request, *response);  // Currently only support one model so hard coded.
```

Comment:
In the `executor.Predict()`, there is some logic to do data type conversion. Could we have one more timer for this line?

https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/server/executor.cc#L104